### PR TITLE
Fix broken links, clean up image dir and update OpenGraph splash

### PR DESCRIPTION
**Pull request by Holzhaus**

### CODE REVIEW

**Comment by Holzhaus** [comment]
Ok, ready to merge!

**Comment by Holzhaus** [comment]
Ping

**Comment by Be-ing** [comment]
For the old links to mixxx.org/irc I think we should either remove the links or change them to link to Zulip. Using archive.org to point people to a web interface for IRC isn't really useful.

**Comment by Holzhaus** [comment]
These are just links in old blog posts that that we keep for historical purposes. I don't think anyone will mind. If you insist I can change them.

**Comment by Be-ing** [comment]
Yes, please change them. I don't mind if you remove them or link to Zulip, but an archive.org link to a web interface for a defunct IRC channel is useless.

**Comment by Holzhaus** [comment]
Done.

**Comment by Holzhaus** [comment]
Merge?

**Comment by Holzhaus** [comment]
Ping

File: pages/features.html [hunk 1]
```diff
@@ -109,15 +109,15 @@ <h2><a class="headline-link" href="#samplers">{% trans "Sampler Decks" %}</a></h
...
+  <img class="responsive drop-shadow" src="{% static '/images/1.11/feature-sampler2.png' %}">
 </div>
 
 <div id="effects" class="docs-section container">
   <h2><a class="headline-link" href="#effects">{% trans "Effects Chains" %}</a></h2>
   <p>
     {% trans "Link up to three effects in a chain for a unique twist on your mix. Fine tune your sound by tweaking each parameter individually. Customize your workflow by assigning different parameters to adjust with the effect's metaknob. Focus different effects from a controller for flexible control of effect chains, whether your controller has a full effects section or a single knob." %}
   </p>
-  <img class="responsive drop-shadow" src="{% static '/static/images/feature_effect_unit_expanded.png' %}">
+  <img class="responsive drop-shadow" src="{% static '/images/1.11/feature-effect-unit-expanded.png' %}">
```
Comment (ronso0) [inline pages/features.html]: I thought `/images/###` refers to the current Mixxx version, so the images in _this_ page should go to `static/images/2.3` ?

**Comment by Holzhaus** [comment]
Merge?

**Pull request merged**